webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala

```diff
@@ -38,6 +38,12 @@ import scala.concurrent.Future
 import scala.util.Failure
 import scala.util.Success
 import scala.util.Try
+import dsp.errors.ValidationException
+import org.knora.webapi.messages.store.triplestoremessages.BooleanLiteralV2
+import dsp.valueobjects.LangString
+
+import dsp.schema.domain.CreatePropertyCommand
+import dsp.schema.domain.{SmartIri => NotAnotherSmartIri}
```

Comment:
I would rename the `NotAnotherSmartIri` to something more professional ;)

Reply:
done